Nodejs/Product/TestAdapter/TestFrameworks/Angular/vsKarmaReporter.js

```diff
@@ -48,7 +48,7 @@ const vsKarmaReporter = function (baseReporterDecorator, config, logger, emitter
         // Increment the amount of test cases found.
         testCaseCount++;
 
-        const fullFilePath = path.join(path.dirname(project.karmaConfigPath), result.fileLocation.relativeFilePath);
+        const fullFilePath = path.join(project.rootPath, result.fileLocation.relativeFilePath);
```

Comment:
I think this is the old value plus `project.root`, but I'd find an example helpful.

Reply:
This will be the `angular.json` file path plus the `project.root` defined on it. I can add a comment.